server/pulp/server/agent/context.py

```diff
@@ -29,9 +30,9 @@ class Context(object):
     some cases DB entity IDs so we can update the DB based on the result of the
     operation on the agent.
 
-    :ivar route: The AMQP route.
-        The route has the form: 'pulp.agent.<consumer_id>'.
-    :type route: str
+    :ivar address: The AMQP address.
+        The address has the form: 'pulp.agent.<consumer_id>'.
+    :type address: str
```

Comment:
Consider using basestring instead of str unless you specifically don't want to allow unicode arguments.